frontend/src/main/scala/akkaviz/frontend/FrontendApp.scala

```diff
@@ -111,12 +111,17 @@ object FrontendApp extends JSApp with Persistence
   private val seenMessages = Var[Set[String]](Set())
   private val selectedMessages = persistedVar[Set[String]](Set(), "selectedMessages")
   private val thrownExceptions = Var[Seq[ActorFailure]](Seq())
+  private val showUnconnected = Var[Boolean](false)
 
-  private val addNodesObs = seenActors.trigger {
+  private val addNodesObs = Rx((showUnconnected(), seenActors())).trigger {
     seenActors.now.foreach {
       actor =>
-        val isDead = deadActors.contains(actor)
-        graph.addNode(actor, js.Dictionary(("dead", isDead)))
+        if(showUnconnected.now || createdLinks.exists(_._1.split("->").contains(actor))) {
```

Comment:
I think it might be very slow if we do it in every graph change iteration, I will try to address that in batched updates.